README.md

```diff
@@ -62,6 +62,8 @@ yarn build
 yarn build:css
 ```
 
+To change settings for you local development environment. Copy your `.env.sample` file to a new file and rename it to `.env.development`. 
```

Comment:
`your` rather than `you`, and I'd suggest `environment, copy` rather than `environment. Copy`